client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -262,7 +262,7 @@ export class DomainDetailsForm extends PureComponent {
 		} else if ( this.needsOnlyGoogleAppsDetails() ) {
 			title = this.props.translate( 'G Suite Account Information' );
 		} else {
-			title = this.props.translate( 'Domain Contact Information' );
+			title = this.props.translate( 'Domain contact information' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Your domain contact information' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)